examples/drag-n-drop/main.js

```diff
@@ -11,29 +11,39 @@ var most = require('most');
 var DROP = 0, GRAB = 1, DRAG = 2;
 
 module.exports = function() {
+	// The area where we want to do the dragging
+	var area = document.querySelector('.dragging-area');
 	// The thing we want to make draggable
 	var draggable = document.querySelector('.draggable');
+	var dragOffset = {};
```

Comment:
Yes, I wasn't much happy with that, but it was the quick easy way to fix the example without changing too much its structure.
Somehow, the reduce call used for its side effect is a bit of a hack, too... :smile: 
As an exercise, I will try and do that in a cleaner way.

I must also remove a console.log() I forgot.